karibu-dsl/src/main/kotlin/com/github/mvysny/karibudsl/v10/KLitRendererTagsDsl.kt

```diff
@@ -0,0 +1,206 @@
+package com.github.mvysny.karibudsl.v10
+
+import com.github.mvysny.karibudsl.v10.KLitRendererBuilderA.Property
+
+@VaadinDsl
+public interface KLitRendererTagsBuilderA<TSource> {
+
+    public operator fun Property<TSource>.unaryPlus()
+    public operator fun KLitRendererBuilderA.Function<TSource>.unaryPlus()
+
+    public operator fun String.unaryPlus()
+
+    public val spacing: KLitRendererTheme get() = KLitRendererTheme.spacing
+    public val padding: KLitRendererTheme get() = KLitRendererTheme.padding
+
+    public fun horizontalLayout(
+        vararg attributes: KLitRendererAttribute,
+        block: KLitRendererTagsBuilderA<TSource>.() -> Unit
+    )
+
+    public fun verticalLayout(
+        vararg attributes: KLitRendererAttribute,
+        block: KLitRendererTagsBuilderA<TSource>.() -> Unit
+    )
+
+    public fun span(
+        vararg attributes: KLitRendererAttribute,
+        block: KLitRendererTagsBuilderA<TSource>.() -> Unit
+    )
+
+    public fun div(
+        vararg attributes: KLitRendererAttribute,
+        block: KLitRendererTagsBuilderA<TSource>.() -> Unit
+    )
+
+    public fun anchor(
+        vararg attributes: KLitRendererAttribute,
+        block: KLitRendererTagsBuilderA<TSource>.() -> Unit
+    )
+
+    public fun icon(
+        vararg attributes: KLitRendererAttribute,
+        block: KLitRendererTagsBuilderA<TSource>.() -> Unit = {}
+    )
+
+    public fun button(
+        vararg attributes: KLitRendererAttribute,
+        block: KLitRendererTagsBuilderA<TSource>.() -> Unit
+    )
+
+    public fun img(
+        vararg attributes: KLitRendererAttribute,
+    )
+
+}
+
+public class KLitRendererTagsBuilder<TSource>(
+
+    private val litRendererBuilder: KLitRendererBuilderA<TSource>,
+
+    private val tagName: String = "",
+    private val attributes: String = "",
+    private val propertyName: String = "",
+
+    /**
+     * --- What are Self Closing Tags in HTML? https://www.scaler.com/topics/self-closing-tags-in-html/
```

Comment:
It's better to update to something like this: by default, all html tags are closed (e.g. `<br/>` or `<b>foo</b>`; set to false in special cases yada yada, please read more on self-closing tags at http:// foo bar